engine/doc/Makefile

```diff
@@ -0,0 +1,208 @@
+# CMAKE generated file: DO NOT EDIT!
+# Generated by "Unix Makefiles" Generator, CMake Version 3.16
+
+# Default target executed when no arguments are given to make.
+default_target: all
```

Comment:
We shouldn't be commiting Makefiles as Cmake is the official source of truth for the build.

Reply:
Sorry. The whole doc folder was downloaded by a merge and I assumed and didn't check it's documentation of the Makefile rather than an actual Makefile. I'll delete it.